### ISSUE TITLE
[45] Enter key listener

### ISSUE THREAD

**Issue by JaRutkowski** [JaRutkowski/eLibrary#149]
Handle enter key action and set the default buttons as below:
- On 'Tabbed' frame (for the first time, after frame will be oppended) - 'Settings' button;
- On 'Clients' tab - 'Add' button;
- On 'Library' tab - 'Add' button on 'Library' panel;
- On 'Books' tab - 'Add' button;
- On 'Loan service' tab - 'Loan' button;
- On 'Dictionaries' tab - 'Add' button;
- On 'Workers' tab - 'Add' button.

The behavior and functionalities should be the same as it is on the 'Login' frame for the 'Log in' button (f.eg. button should be highlighted). 

Sample screen for the 'Clients' tab:
![image](https://user-images.githubusercontent.com/26207880/78231050-84e86b80-74d2-11ea-9db6-d4e0e9f583aa.png)

Basically from technical point of view it should be done like below:
`getFrame().getRootPane().setDefaultButton(panelClient.getCockpitEditionPanel().getBtnAdd());`

The issue is similar to #121.
